packages/eslint-plugin/src/rules/keyword-spacing.ts

```diff
@@ -0,0 +1,598 @@
+import keywords from 'eslint/lib/rules/utils/keywords';
+import * as util from '../util';
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { JSONSchema4 } from 'json-schema';
+import { isTokenOnSameLine, isKeywordToken } from '../util/astUtils';
+import { isNotOpeningParenToken } from 'eslint-utils';
+
+export type Option = Partial<{
+  before: boolean;
+  after: boolean;
+}>;
+
+export type RootOption = Option & {
+  overrides?: { [keywordName: string]: Option };
+};
+
+export type Options = [RootOption];
+export type MessageIds =
+  | 'expectedBefore'
+  | 'expectedAfter'
+  | 'unexpectedBefore'
+  | 'unexpectedAfter';
+
+//------------------------------------------------------------------------------
+// Constants
+//------------------------------------------------------------------------------
+
+const PREV_TOKEN = /^[)\]}>]$/u;
+const NEXT_TOKEN = /^(?:[([{<~!]|\+\+?|--?)$/u;
+const PREV_TOKEN_M = /^[)\]}>*]$/u;
+const NEXT_TOKEN_M = /^[{*]$/u;
+const TEMPLATE_OPEN_PAREN = /\$\{$/u;
+const TEMPLATE_CLOSE_PAREN = /^\}/u;
+const CHECK_TYPE = /^(?:JSXElement|RegularExpression|String|Template)$/u;
+const KEYS = keywords.concat([
```

Comment:
I could hardcode the list of keyword here instead of importing it from `eslint`.

Reply:
Importing them is completely fine :)